test/SymmetricPluginManagerTest.php

```diff
@@ -44,6 +45,9 @@ public function testHas($symmetric)
     public function testGet($symmetric)
     {
         $plugin = new SymmetricPluginManager();
+        if (! extension_loaded($symmetric)) {
```

Comment:
@samsonasik why? The symmetric plugins are loaded only on the `get()` execution, not in the constructor of `SymmetricPluginManager`.